cdk/src/functions/service-control-policies-root.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  AttachPolicyCommand,
+  CreatePolicyCommand,
+  DeletePolicyCommand,
+  DetachPolicyCommand,
+  ListRootsCommand,
+  OrganizationsClient,
+  PolicyType,
+  UpdatePolicyCommand,
+  ListPoliciesCommand,
+} from '@aws-sdk/client-organizations';
+import { CdkCustomResourceEvent, CdkCustomResourceResponse, Context } from 'aws-lambda';
+
+async function getRootId(organizationClient: OrganizationsClient): Promise<string | undefined> {
+  let id = 'error'; //set to error. Update if root ID is found. Then, update it to Root Id.
```

Comment:
This is not a clean style. I think we should just throw an exception if there is not root OU.